packages/openapi-pinia-colada/test/fixtures/mock-server.ts

```diff
@@ -0,0 +1,121 @@
+import {
+  http,
+  HttpResponse,
+  type JsonBodyType,
+  type StrictRequest,
+  type DefaultBodyType,
+  type HttpResponseResolver,
+  type PathParams,
+  type AsyncResponseResolverReturnType,
+} from "msw";
+import { setupServer } from "msw/node";
```

Comment:
(Informational; nothing to change) Just a note on msw vs standard `fetch` mocking—we’ve gone back-and-forth on how to best test the fetch wrappers, and you’ll see a little blend of both in the test suites. Really, the short answer is “use whatever’s easiest.” But for openapi-fetch specifically, we were seeing msw’s really heavy mocking interfere with our performance benchmarks, so we started using less of it.

Though it does cause some delay in testing, it’s not perceptible to make a difference. Our perf benchmarks were the real reason for it.

As long as you’re not **mocking any internal part of the library,** pretty much anything goes—whether that means mocking `fetch`, or mocking Node’s fetch, or even talking to a local Express server, etc. It’s up to you and what makes sense. But since this is Vue-based, soft recommendation to follow what other similar libraries do, so Vue contributors have an easier time contributing to this lib.